lib/rules/no-invalid-model-keys.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * @fileoverview Requires valid keys in model property.
+ * @author Alex Sokolov
+ */
+'use strict'
+
+const utils = require('../utils')
+
+/**
+ * @typedef {import('../utils').GroupName} GroupName
+ */
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+/** @type {GroupName[]} */
+const GROUP_NAMES = ['model']
+
+const VALID_MODEL_KEYS = ['prop', 'event']
+
+module.exports = {
+  meta: {
+    type: 'problem',
+    docs: {
+      description: 'require valid keys in model prop',
+      categories: ['vue3-recommended', 'recommended'],
```

Comment:
Do not change the shareable configuration until a major version upgrade. Would you please remove `meta.docs.categories`.

```suggestion
      categories: undefined,
```